examples/ota-provider-app/linux/main.cpp

```diff
@@ -300,7 +308,9 @@ OptionSet cmdLineOptions = { HandleOptions, cmdLineOptionsDef, "PROGRAM OPTIONS"
                              "  -x, --ignoreQueryImage <ignore count>\n"
                              "        The number of times to ignore the QueryImage Command and not send a response.\n"
                              "  -y, --ignoreApplyUpdate <ignore count>\n"
-                             "        The number of times to ignore the ApplyUpdateRequest Command and not send a response.\n" };
+                             "        The number of times to ignore the ApplyUpdateRequest Command and not send a response.\n"
+                             "  -P, --pollFrequency <frequency in milliseconds>\n"
```

Comment:
Please also update the [README](https://github.com/project-chip/connectedhomeip/tree/master/examples/ota-provider-app/linux#usage)

Reply:
> Please also update the [README](https://github.com/project-chip/connectedhomeip/tree/master/examples/ota-provider-app/linux#usage)

Done.